specs/endpoint-security.spec.yml

```diff
@@ -90,3 +88,26 @@ inputs:
       log:
         path: "C:\\Program Files\\Elastic\\Endpoint\\state\\log\\endpoint-*.log"
       operations: *operations
+  - name: endpoint
+    description: "Endpoint Security"
+    platforms:
+      - container/amd64
+      - container/arm64
+    outputs:
```

Comment:
best to use a yaml reference for outputs as they look to be the same

Reply:
Will do! Thanks for the suggestion.